internal/command/jsonformat/change/change.go

```diff
@@ -60,6 +61,11 @@ func (change Change) Warnings(indent int) []string {
 	return change.renderer.Warnings(change, indent)
 }
 
+// GetAction returns the plans.Action that this change describes.
+func (change Change) GetAction() plans.Action {
```

Comment:
```suggestion
func (change Change) Action() plans.Action {
```

Just an opinionated style nit. Alternatively, we could consider making the field public.

Reply:
I've added this to the set of general refactors, I'll do at the end.

Changing it here and now will make merging the rest of the PRs more difficult because of merge conflicts